### PR TITLE
Update libs and anchor solana config for client

### PR DESCRIPTION
**Pull request by jowlee**
### Description
Update libs with the audius data anchor config

### How Has This Been Tested?
Ran locally and was able to create and init user, create tracks, playlist, social actions, etc. in the console

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "version": "1.1.6",
   "private": true,
   "dependencies": {
-    "@audius/libs": "1.2.114",
+    "@audius/libs": "^1.2.115",
```
Comment (dylanjeffers) [inline packages/web/package.json:9]: prob pin?

**Review by dylanjeffers — APPROVED** [review]
awesome!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-anchor-solana-libs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-anchor-solana-libs

**Pull request merged**